inc/umock_c/umock_c_internal.h

```diff
@@ -4,6 +4,14 @@
 #ifndef UMOCK_C_INTERNAL_H
 #define UMOCK_C_INTERNAL_H
 
+#ifdef __cplusplus
+#include <cstdlib>
+#include <cstdio>
+#else
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
>stdio [](start = 10, length = 5)

there's at least 1 size_t in the file, so that warrants a stddef.

there's a fair amount of memcpy, that is string.h.

But I cannot see at a simple glance what is needed from stdlib or stdio.

Reply:
stdlib.h has size_t.

7.20 General utilities <stdlib.h>
1 The header <stdlib.h> declares five types and several functions of general utility, and
defines several macros.257)
2 The types declared are size_t and wchar_t (both described in 7.17),
div_t
which is a structure type that is the type of the value returned by the div function

---
In reply to: [293559482](https://github.com/Azure/umock-c/pull/95#discussion_r293559482) [](ancestors = 293559482)